views/calendar/calendar.js

```diff
@@ -0,0 +1,416 @@
+                send an <a href="https://www.wikitree.com/wiki/Category:E-Cards">e-card</a>.`,
+            docs: "",
+        };
+    }
+    init(selector, person_id) {
+        WikiTreeAPI.postToAPI({
+            action: "getPerson",
+            key: person_id,
+            fields: "Name",
+        }).then(function (data) {
+            var AllMonths = {
+                Jan: [],
+                Feb: [],
+                Mar: [],
+                Apr: [],
+                May: [],
+                Jun: [],
+                Jul: [],
+                Aug: [],
+                Sep: [],
+                Oct: [],
+                Nov: [],
+                Dec: [],
+            };
+
+            /* The default view for the Calendar is a vertical display of events from the logged in
+             * user's watchlist. This is based on the original Special:Anniversaries view.
+             */
+            $(".ancCalendar").remove();
+            $(selector).append(`
+                <div class="watchCalendar">
+                    <div class="fourteen columns center">
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Jan">January</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Feb">February</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Mar">March</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Apr">April</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-May">May</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Jun">June</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Jul">July</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Aug">August</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Sep">September</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Oct">October</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Nov">November</span>&nbsp;
+                        &nbsp;<span style="text-decoration:underline;cursor:pointer;" id="month-Dec">December</span>&nbsp;
+                    </div><br>
+                    <div class="loader">
+                        <image width="200px" src="views/calendar/wtloadericon.png" />
+                    </div>
+                    <div class="calendar">
+                        <div id="Jan" class="content"><h2>January</h2></div>
+                        <div id="Feb" class="content"><h2>February</h2></div>
```

Comment:
I've noticed, there are several places using month names (either strings or objects)

Maybe better way is to define constant with array of those names and then just use it as a base in combination with map and join functions.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

You can reduce those names to 3 letters using map and slice functions

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice

As for array to object, check this

https://stackoverflow.com/a/36388401